src/server/json-hash.ts

```diff
@@ -15,7 +15,7 @@ type ResponseResult<LoaderData> = {
 export async function jsonHash<LoaderData extends Record<string, unknown>>(
 	input: LoaderData,
 	init?: ResponseInit | number,
-): Promise<TypedResponse<ResponseResult<LoaderData>>> {
+): Promise<UNSAFE_DataWithResponseInit<ResponseResult<LoaderData>>> {
```

Comment:
I think this could be left for TS to infer it

Reply:
Yeah I agree, especially now that TypedResponse got nuked out of existence